Utils/Dataflow/091_datasetsRucio/README

```diff
@@ -72,4 +72,14 @@ example:
 	"taskid": 8343552
 }
 
+5. Samples
+----------
+The 'output' directory contains two pairs of samples:
```

Comment:
I think it might be good to explain, what these samples are: 
* 'input_*' with task metadata, extended with `primary_input` dataset metadata;
* 'output_*' with output datasets metadata.

Of course the rest of the README gives no clue about "how to get input/output ds metadata", but we were going to  fix it (sooner or later) in a separate PR..

Reply:
Done.

Yes, this README could use an overhaul. Apart from what you said, I'm not sure that Input and Output parts are needed due to presence of the samples, possible changes in them, and confusion with input/output (datasets / of the stage).